### PR TITLE
Link to release description on downloads.dd

### PR DESCRIPTION
**Pull request by wilzbach**
Cherry-picked from #1694 as this was controversial:

> Not sure about making the platform name a link to the documentation. I'd think that being on the 
download page, clicking it would take me to all Windows downloads.

We could also add a link "Docs", but I though that this is the leaner way to do it.

### CODE REVIEW

**Comment by CyberShadow** [comment]
Hmm, how about something like show an 🛈 on hover next to the platform name that links to the help page or something like that?

**Comment by wilzbach** [comment]
> Hmm, how about something like show an 🛈 on hover next to the platform name that links to the help page or something like that?

Before:

image

After:

image

**Comment by CyberShadow** [comment]
```css
.download_paragraph > span {
    visibility: hidden;
    font-size: 125%;
}
.download_paragraph:hover > span {
    visibility: visible;
}
```
?

**Comment by wilzbach** [comment]
Well I like the increased font-size, but I personally don't like user interfaces on which I don't immediately see all buttons, but I don't feel strongly about this & would like to move forward here
-> I added your CSS definitions -> should be good to go?

**Comment by CyberShadow** [comment]
Not everything on the page is equally important, so bombarding visitors with visual clutter won't result in the best experience. CSS hover tricks are probably not the best way to avoid that, but I think it's still better than nothing.

**Comment by MartinNowak** [comment]
I've first taken this for a security warning or sth. Looks definitely more like an alert than an info. Let's better change the font color to blue or black.

File: download.dd [hunk 1]
```diff
@@ -64,17 +64,29 @@ $(DIVC download_channels,
...
+        $(SBTN $(ARCH osx, tar.xz), tar.xz)
+    )
 
-    $(DOWNLOAD $(UBUNTU) &nbsp; $(DEBIAN), Ubuntu/Debian, $(SBTN $(DEB32), i386) $(SBTN $(DEB64), x86_64) $(SBTN $(ARCH linux, tar.xz), tar.xz))
+    $(DOWNLOAD Ubuntu/Debian, $(UBUNTU) &nbsp; $(DEBIAN), linux,
+        $(SBTN $(DEB32), i386) $(SBTN $(DEB64), x86_64) $(SBTN $(ARCH linux, tar.xz), tar.xz)
+    )
 
-    $(DOWNLOAD $(FEDORA) &nbsp; $(CENTOS), Fedora/CentOS, $(SBTN $(RPM32), i386) $(SBTN $(RPM64), x86_64) $(SBTN $(ARCH linux, tar.xz), tar.xz))
+    $(DOWNLOAD Fedore/CentOS, $(FEDORA) &nbsp; $(CENTOS), linux,
```
Comment (CyberShadow) [inline download.dd]: It's "Fedora", not "Fedore".

**Comment by CyberShadow** [comment]
-> https://github.com/dlang/dlang.org/pull/2059

**Pull request merged**